packaging/cross_build.sh

```diff
@@ -113,7 +113,7 @@ echo "Successfully built the binary: ${TARGET_BIN}"
 # linking check
 echo "Printing linking information for the binary..."
 file "$TARGET_BIN"
-ldd "$TARGET_BIN"
+ldd "$TARGET_BIN" || true
```

Comment:
Ok, this definitely needs at least a comment!

Reply:
It would, I have no idea why this was there :D the good news it it isn't needed so I'll just remove it